client/keys.go

```diff
@@ -54,11 +62,34 @@ func NewDiscoveryKeysAPI(c HTTPClient) KeysAPI {
 }
 
 type KeysAPI interface {
-	Create(ctx context.Context, key, value string, ttl time.Duration) (*Response, error)
+	Set(ctx context.Context, key, value string, opts *SetOptions) (*Response, error)
+	Create(ctx context.Context, key, value string) (*Response, error)
```

Comment:
understand. `Create` and `Update` just does the minimal work, and `Set` gives the full features.